go.mod

```diff
@@ -144,10 +144,10 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	gopkg.in/yaml.v2 v2.4.0
 	helm.sh/helm/v3 v3.9.0
-	k8s.io/api v0.26.1
-	k8s.io/apiextensions-apiserver v0.25.12
-	k8s.io/apimachinery v0.26.1
-	k8s.io/apiserver v0.25.12
+	k8s.io/api v0.27.2
+	k8s.io/apiextensions-apiserver v0.27.2
+	k8s.io/apimachinery v0.27.2
+	k8s.io/apiserver v0.27.2
```

Comment:
I don't think it is possible to bump these deps, at least for now?

Reply:
fixed